.github/workflows/conda-package.yml

```diff
@@ -88,13 +97,22 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-python-${{ matrix.python }}-
             ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-
+      - name: Store conda paths as envs
+        shell: bash -l {0}
+        run: |
+          echo "WHEELS_OUTPUT_FOLDER=$GITHUB_WORKSPACE${{ runner.os == 'Linux' && '/' || '\\' }}" >> $GITHUB_ENV
       - name: Build conda package
         run: conda build --no-test --python ${{ matrix.python }} -c intel -c conda-forge --override-channels conda-recipe
       - name: Upload artifact
         uses: actions/upload-artifact@v3
         with:
           name: ${{ env.PACKAGE_NAME }} ${{ runner.os }} Python ${{ matrix.python }}
           path: ${{ env.conda-bld }}${{ env.PACKAGE_NAME }}-*.tar.bz2
+      - name: Upload wheels artifact
+        uses: actions/upload-artifact@v3.1.2
```

Comment:
Is there a reason why not use `actions/upload-artifact@v3` like it is done in the previous "Upload artifact" step?

Reply:
No, `v3` is absolutely fine!